src/common/navigation.yaml

```diff
@@ -7,11 +7,11 @@ sections:
           - path: /overview
           - path: /testnet
           - path: /regtest
-          - path: /proof-of-transfer
-          - path: /mining
           - path: /accounts
           - path: /transactions
           - path: /network
+          - path: /proof-of-transfer
+          - path: /mining
```

Comment:
A bit unrelated to the issue, but I noticed that we have 3 docs that start with "understand [...]". It seems duplicated with the content being in the "understand stacks" section.

Maybe we could change it up for consistency?
- Understanding Mining > Mining
- Understand Stacking > Stacking
- Understanding the Stacks CLI > Stacks CLI

Feel free to park this if you believe it is out of scope

Reply:
Agree, let's just make that change now.